examples/blend/main.rs

```diff
@@ -148,9 +148,8 @@ impl<B: gfx::Backend> gfx_app::Application<B> for App<B> {
         }
     }
 
-    fn render<Gp>(&mut self, (frame, semaphore): (gfx::Frame, &gfx::handle::Semaphore<B::Resources>),
-                  pool: &mut Gp, queue: &mut gfx::queue::GraphicsQueueMut<B>)
-        where Gp: gfx::GraphicsCommandPool<B>
+    fn render(&mut self, (frame, semaphore): (gfx::Frame, &gfx::handle::Semaphore<B::Resources>),
+              pool: &mut gfx::GraphicsCommandPool<B>, queue: &mut gfx::queue::GraphicsQueueMut<B>)
```

Comment:
I think this would be possible, `shadow` requires additional command pools, which should be created at initialization and probably some additional synchronization.